src/components/tables/RowActions/Delete/Progress.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, CircularProgress, ListItemText, Stack } from '@mui/material';
+import { createEntityDraft } from 'api/drafts';
+import { createDraftSpec } from 'api/draftSpecs';
+import { createPublication } from 'api/publications';
+import Error from 'components/shared/Error';
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    deleting: any;
```

Comment:
Would either `entity`, `entityToDelete`, or `entityTarget` work as alternative property names? Something about `deleting` is a little confusing for me.

Reply:
I refactored stuff so the "shared" version of this takes in `entity`